run_rectests.sh

```diff
@@ -56,6 +56,12 @@ tests=(
 temp_dir=$(mktemp -d)
 trap 'rm -rf "$temp_dir"' EXIT
 
+interrupt() {
+    echo "Test run interrupted" >&2
```

Comment:
does this also do the rm-rf above?

Reply:
That is run yeah, no temp dir left behind.